databuilder/models/table_metadata.py

```diff
@@ -99,13 +138,13 @@ def __init__(self,
         :param source: The unique source of what is populating this description.
         :param text: the description text. Markdown supported.
         """
-        self._source = source
-        self._text = text
+        self.source = source
```

Comment:
rds models do not have a generic description model. There is only description model for each entity(e.g., TableDescription, SchemaDescription). The code change is going to create description table records from databuilder Table/Schema/Column models, namely, access some attributes outside of DescriptionMetadata class. Should I change it back to protected vars?(I think it may bring some syntax highlight warning). Thanks